libs/service-portal/education-career/src/screens/EducationCareer/EducationCareer.tsx

```diff
@@ -18,7 +18,6 @@ function EducationCareer(): JSX.Element {
           defaultMessage: 'Hér getur þú fundið yfirlit yfir námsmat.',
           description: 'education career intro',
         })}
-        img="./assets/images/educationGrades.svg"
```

Comment:
has the image been removed as well?

Reply:
Yes all images on these landing screens are removed